tests/testing/unit/assertionTemplate.tsx

```diff
@@ -142,6 +161,24 @@ describe('assertionTemplate', () => {
 		h.expect(childAssertion);
 	});
 
+	it('can remove a node in the root', () => {
+		const h = harness(() => w(MultiRootWidget, { last: false }));
+		const insertionAssertion = baseMultiRootAssertion.remove('~last');
+		h.expect(insertionAssertion);
+	});
+
+	it('can remove a node', () => {
```

Comment:
@aciccarello this test seems to be duplicated and is causing intern to get upset... the exact same test is two tests above, could have been accidentally left there when merging a conflicts perhaps?

Reply:
@agubler As I look at that more, I think that was meant to be a replace node test. Sorry. 😞